### ISSUE TITLE
The upload button fails contrast ratio on hover state for non-text contrast

### ISSUE THREAD

**Issue by jose-cds** [cds-snc/platform-forms-client#3099]
**HC Assessment label: Moderate**

Pages:  Advancing Accessibility Standards Research Program Expression of Interest
URL: https://forms-staging.cdssandbox.xyz/en/id/134
User agent:  Chrome (v119.0.6045.200)
Language: English and French

Issue Description: 
The "Upload a file" button fails contrast ratio on hover state against its adjacent colour for non-text contrast.
Hover state = rgb(160, 174, 192) = #A0AEC0
Adjacent colour = #FFFFFF

Steps to reproduce:

1. Open the above URL in the Chrome browser
2. Using the mouse, hover over the "Upload a file" button
3. Use the the Colour Contrast Analyser tool to verify the contrast ratio

Expected result: The button in question should have a contrast ratio of at least 3:1 against its adjacent colour on hover state.

**Comment by srtalbot** [comment]
We removed the file upload feature. We will reassess this fiscal year.
